### DISCUSSION TITLE
Added: Notes for September 23rd, 2020

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#3943] [LDM Notes]
https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-09-23.md

https://github.com/dotnet/csharplang/pull/3936

Agenda
* General improvements to the `struct` experience

**Comment by jnm2** [comment]
@333fred The linked filename is correct but it contains "September 14th" in the title.

**Comment by Joe4evr** [comment]
So once the decisions for these Low Level Struct Improvements have been made, is there a decent chance of following up with some *High* Level Struct Improvements in the not-too-distant future?

Specifically calling out #99 and #146 as things that devs have been wanting for years.

**Reply by huoyaoyuan** [reply]
I think non-defaultable struct should be a must if ref fields comes. Further more, the newer runtime could throw when getting default on a struct that declares to deny it.
